ClassLibrary1/Tests.cs

```diff
@@ -0,0 +1,112 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using Excel = Microsoft.Office.Interop.Excel;
+
+namespace Tests
+{
+    [TestFixture]
+    public class Tests
+    {
+        public const string url = "https://www.tut.by/";
+        public const string username = "seleniumtests@tut.by";
+        public const string password = "123456789zxcvbn";
+
+        public const string frameUrl = "https://the-internet.herokuapp.com/iframe";
+        public const string alertUrl = "https://the-internet.herokuapp.com/javascript_alerts";
+
+        [Test]
+        public void Login()
+        {
+            var driver = Drivers.Methods.NavigateTo(url);
+            var actualUrl = Drivers.Methods.GetUrl(driver);
+            Assert.IsTrue(actualUrl.StartsWith(url), "invalid url)");
+
+            Drivers.Methods.PopulateLoginForm(driver, username, password);
+
+            var submitButtonState = Drivers.Methods.IsLoginButtonEnabled(driver);
+            Assert.IsTrue(submitButtonState);
+
+            Drivers.Methods.LoginUser(driver);
+
+            var loginState = Drivers.Methods.IsUserLogin(driver);
+            
+            Assert.IsTrue(loginState);
```

Comment:
You can combine these last two lines

Reply:
Changes are made in the new commit under task 70